web/packages/teleport/src/Account/ManageDevices/AddDevice/AddDevice.tsx

```diff
@@ -191,47 +201,51 @@ export function AddDevice({
-                  width="100%"
-                  label="Two-factor Type"
-                  data-testid="mfa-select"
-                  value={mfaOption}
-                  options={mfaOptions}
-                  onChange={(o: MfaOption) => {
-                    validator.reset();
-                    onSetMfaOption(o);
-                  }}
-                  mr={3}
-                  isDisabled={addDeviceAttempt.status === 'processing'}
-                  elevated={true}
-                />
-                {mfaOption.value === 'otp' && (
-                  <FieldInput
-                    width="50%"
-                    label="Authenticator Code"
-                    rule={requiredToken}
-                    inputMode="numeric"
-                    autoComplete="one-time-code"
-                    value={otpToken}
-                    onChange={e => setOtpToken(e.target.value)}
-                    placeholder="123 456"
-                    readonly={addDeviceAttempt.status === 'processing'}
-                  />
-                )}
-                {mfaOption.value === 'webauthn' && isPasswordlessEnabled && (
+              {restrictDeviceUsage !== 'passwordless' && (
+                <Flex alignItems="center">
                   <FieldSelect
-                    width="50%"
-                    label="Allow Passwordless Login?"
-                    value={usageOption}
-                    options={deviceUsageOpts}
-                    onChange={(o: DeviceusageOpt) => setUsageOption(o)}
+                    maxWidth="50%"
+                    width="100%"
+                    label="Two-factor Type"
+                    data-testid="mfa-select"
+                    value={mfaOption}
+                    options={mfaOptions}
+                    onChange={(o: MfaOption) => {
+                      validator.reset();
+                      onSetMfaOption(o);
+                    }}
+                    mr={3}
                     isDisabled={addDeviceAttempt.status === 'processing'}
                     elevated={true}
                   />
-                )}
-              </Flex>
+                  {mfaOption.value === 'otp' && (
+                    <FieldInput
+                      width="50%"
+                      label="Authenticator Code"
+                      rule={requiredToken}
+                      inputMode="numeric"
+                      autoComplete="one-time-code"
+                      value={otpToken}
+                      onChange={e => setOtpToken(e.target.value)}
+                      placeholder="123 456"
+                      readonly={addDeviceAttempt.status === 'processing'}
+                    />
+                  )}
+                  {mfaOption.value === 'webauthn' &&
+                    isPasswordlessEnabled &&
+                    !restrictDeviceUsage && (
+                      <FieldSelect
+                        width="50%"
+                        label="Allow Passwordless Login?"
```

Comment:
i'm curious, why we leave it as a option? (like why not remove it entirely?)

Reply:
@kimlisa It's just because the old code is still around (the change that disabled the old one was not merged when I was sending this one). I'll address this in a cleanup PR, per that TODO that I left in one of the files.